Sources_v3/StreamChatUI/Generated/L10n.swift

```diff
@@ -10,125 +14,139 @@ import Foundation
 // swiftlint:disable explicit_type_interface function_parameter_count identifier_name line_length
 // swiftlint:disable nesting type_body_length type_name vertical_whitespace_opening_braces
 internal enum L10n {
-
-  internal enum Alert {
-    internal enum Actions {
-      /// Cancel
-      internal static let cancel = L10n.tr("Localizable", "alert.actions.cancel")
-      /// Delete
-      internal static let delete = L10n.tr("Localizable", "alert.actions.delete")
-    }
-  }
-
-  internal enum Channel {
-    internal enum Name {
-      /// and
-      internal static let and = L10n.tr("Localizable", "channel.name.and")
-      /// NoChannel
-      internal static let missing = L10n.tr("Localizable", "channel.name.missing")
-      /// more
-      internal static let more = L10n.tr("Localizable", "channel.name.more")
-    }
-  }
-
-  internal enum Composer {
-    internal enum Checkmark {
-      /// Also send in channel
-      internal static let channelReply = L10n.tr("Localizable", "composer.checkmark.channel-reply")
-      /// Also send as direct message
-      internal static let directMessageReply = L10n.tr("Localizable", "composer.checkmark.direct-message-reply")
-    }
-    internal enum Picker {
-      /// Cancel
-      internal static let cancel = L10n.tr("Localizable", "composer.picker.cancel")
-      /// File
-      internal static let file = L10n.tr("Localizable", "composer.picker.file")
-      /// Photo
-      internal static let image = L10n.tr("Localizable", "composer.picker.image")
-      /// Choose attachment type: 
-      internal static let title = L10n.tr("Localizable", "composer.picker.title")
+    internal enum Alert {
+        internal enum Actions {
+            /// Cancel
+            internal static let cancel = L10n.tr("Localizable", "alert.actions.cancel")
+            /// Delete
+            internal static let delete = L10n.tr("Localizable", "alert.actions.delete")
+        }
     }
```

Comment:
Please only include changes related to your PR, the rest is just reformatting.

Reply:
Well, I think I couldn't affect this because I added new localization and the change triggered the Localization generation. Maybe we could do something about this? I was also a bit worried about the change of the whole file just because of generation of one Localizable... We can discuss this?

Reply:
Fixed.